### DISCUSSION TITLE
What is the benefits of ToBase64Trimmed

### DISCUSSION THREAD

**Discussion by YousefAK009** [alirezanet/InfiniteEnumFlags#8] [Q&A] [answered]
I see it in the source code and I don't know if it should to use it or not

**Comment by alirezanet** [comment, marked as answer]
by default you should not use this.
 
It is the same as base64 string ... but It removes the empty bits before generating the base 64 string.

for example imagine this is your enum bits: `00000000100000000000000000`, this method converts the array to a shorter version if it is possible but you will lose the length.  `000000001`.  it is suitable for test purposes.

Generally, I don't recommend using the base64 formats, since `UniqueId` is more consistent.

**Reply by YousefAK009** [reply]
Big thanks to you and your library 🙂
